source/_integrations/nordpool.markdown

```diff
@@ -37,9 +34,21 @@ Currency:
   description: Currency to display prices in. EUR is the base currency in Nord Pool prices.
 {% endconfiguration_basic %}
 
+{% tip %}
+Only a single configuration entry is supported. Use the reconfigure option from the configuration entry if needed to modify the settings.
+
+EUR is the base currency for market prices. If you choose another currency, you can find the conversion rate in the `Exchange rate` sensor.
+All prices are displayed as `[Currency]/kWh`.
+{% endtip %}
+
+## Data fetching and limitations
+
+Data is polled from the Nord Pool API on an hourly basis, exactly on the hour to always ensure the price sensors are displaying the correct price.
+In the case a polling can not happen, because there is no connectivity or a malfunctioning API, there is no retry and the next periodic update will try again.
```

Comment:
```suggestion
Data is polled from the Nord Pool API on an hourly basis, exactly on the hour, to ensure the price sensors are displaying the correct price.
If polling can not happen because of no connectivity or a malfunctioning API, there is no retry; the next periodic update will try again.
```